.github/workflows/onRelease.yml

```diff
@@ -0,0 +1,18 @@
+# when a github release happens, publish an npm package,
+on:
+  release:
+    types: [released]
+  # support manual release
+  workflow_dispatch:
+    inputs:
```

Comment:
Could allow passing in [dryrun](https://github.com/salesforcecli/github-workflows/blob/main/.github/workflows/npmPublish.yml#L14) and [prerelease](https://github.com/salesforcecli/github-workflows/blob/main/.github/workflows/npmPublish.yml#L19) inputs so that they can be passed through from the GitHub UI when manually kicking this off.

Reply:
`types: [prereleased]` is a separate gh event that we would trigger a release off of.  We'll get to explore that for sfdx-core (v3 branch).

I couldn't think of a use case for dryrun here, but some repo will probably have it eventually.